proptest/src/test_runner/config.rs

```diff
@@ -217,7 +223,9 @@ pub struct Config {
     /// See the docs of [`FileFailurePersistence`](enum.FileFailurePersistence.html)
     /// and [`MapFailurePersistence`](struct.MapFailurePersistence.html) for more information.
     ///
-    /// The default cannot currently be overridden by an environment variable.
+    /// You can disable failure persistence with the `PROPTEST_DISABLE_FAILURE_PERSISTENCE`
+    /// environment variable but its not currently possible to set the persistence file
+    /// with an environment variable.
```

Comment:
There was recently a PR adding a comment to these saying "the env var only has an effect when the `std` feature is enabled". I'm assuming that applies here? If so, should we add that warning?

Reply:
Updated